### PR TITLE
Added Scholars Portal Journals translator.

### PR DESCRIPTION
**Pull request by bkawula**

### CODE REVIEW

**Comment by adam3smith** [comment]
Looks great, thanks

File: Scholars Portal Journals.js [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+function detectWeb(doc, url) { 
+	var myArticles = doc.getElementById('my-articles');
+	if (myArticles) {
+  		Zotero.monitorDOMChanges(myArticles.parentElement, {attributes: true, attributeFilter: ["style"]});
+	}
+	// see if saved list is toggled open
+	if (doc.getElementsByClassName('inner-wrap-open') [0]) {
+		if (getItems(doc, true)) {
+			return 'multiple'
```
Comment (zuphilip) [inline Scholars Portal Journals.js]: Missing semicolon

File: Scholars Portal Journals.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+		}
+	} else if (url.indexOf("/search?q") != -1 || url.indexOf("/browse/") != -1) {
+		if (getItems(doc, true)) {
+			return 'multiple'
```
Comment (zuphilip) [inline Scholars Portal Journals.js]: Missing semicolon

File: Scholars Portal Journals.js [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+
+function doWeb(doc, url) {
+	var type = detectWeb(doc, url);
+	if(type == "multiple") {
+		var list = getItems(doc, url);
+		Zotero.selectItems(list, function(selectedItems) {
+			if(!selectedItems) return true;
+			var articles = [];
+			for(var i in selectedItems) {
+				var article = '/ris?uri='+i
```
Comment (zuphilip) [inline Scholars Portal Journals.js]: Missing semicolon

File: Scholars Portal Journals.js [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+				articles.push(article);
+			}
+			ZU.doGet(articles, scrape);
+		})
```
Comment (zuphilip) [inline Scholars Portal Journals.js]: Missing semicolon

File: Scholars Portal Journals.js [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+	} else {
+		var uri = getURI(url);
+		var article = '/ris?uri='+uri
```
Comment (zuphilip) [inline Scholars Portal Journals.js]: Missing semicolon

File: Scholars Portal Journals.js [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+		ZU.doGet(article, scrape);
+	}
+}
+
+function getURI(url){
+	if(url.indexOf("xml") != -1){
+		var a = url.indexOf("details");
+		var b = url.indexOf("xml");
+		uri = url.substring(a+7,b+3);
+		return uri
```
Comment (zuphilip) [inline Scholars Portal Journals.js]: Missing semicolon

File: Scholars Portal Journals.js [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+	} else if(url.indexOf("resolver.scholarsportal.info/resolve/") != -1)  {
+		uri = url.split("resolver.scholarsportal.info/resolve")[1] + ".xml";
+		return uri
```
Comment (zuphilip) [inline Scholars Portal Journals.js]: Missing semicolon

File: Scholars Portal Journals.js [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+		var uri = getURI(item.attachments[0].path);
+		var pdfURL = "/pdf" + uri;
+		item.url = "http://journals.scholarsportal.info/details" + uri;
+		item.attachments = [{
+			url: pdfURL,
+			title: "Scholars Portal Full Text PDF",
+			mimeType: "application/pdf"
+		}];
+		item.complete();
+	})
```
Comment (zuphilip) [inline Scholars Portal Journals.js]: Missing semicolon

**Pull request merged**